src/main/java/Filter.java

```diff
@@ -3,5 +3,5 @@
  * based on given specifications.
  */
 public interface Filter {
-    public boolean checkSchedule(Schedule s);
+    public Schedule checkSchedule(Schedule s);
 }
```

Comment:
Make sure to apply spotless because there should be newline at the end of every file